nuntium/static/js/who.js

```diff
@@ -0,0 +1,87 @@
+$(function(){
+
+  // Copy the "uncontactable" class from the original dropdown `<option>`s,
+  // onto the chosen.js dropdown `<li>`s.
+  $('.chosen-person-select').on('chosen:showing_dropdown', function(e, params){
+    var $options = $(params.chosen.dropdown[0]).find('li');
+    $options.each(function(i){
+      var originalOptElement = params.chosen.results_data[i];
+      if(originalOptElement.classes.indexOf('uncontactable') !== -1){
+        $(this).addClass('uncontactable');
+      }
+    });
+  });
+
+  var showUncontactableWarning = function showUncontactableWarning($selectedOption, $originalSelect, $chosenSelect){
+    var $warning = $('<div>').addClass('alert alert-danger uncontactable-alert');
+    var warningSentence = interpolate(
+      gettext("We do not have contact details for %s."),
```

Comment:
We use “do not” here, but “don’t” on the `missing_contacts.html` page. Maybe we should standardise? How conversational is the rest of the WiP UI?